packages/driver/src/cy/assertions.ts

```diff
@@ -324,6 +326,10 @@ export const create = (Cypress, cy) => {
       // bail if we have no assertions and apply
       // the default assertions if applicable
       if (!cmds.length) {
+        cy.state('overrideAssert', function (...args) {
```

Comment:
how does is this handled when [cmdHadFuncArg](https://github.com/cypress-io/cypress/blob/bc06bdc68de0dea448e782c58f9d3a5dc762843d/packages/driver/src/cy/assertions.ts#L442)?

I see the default behavior checks for a `cy.state('overrideAssert')` function.

Reply:
@BlueWinds Could you add a comment here explaining why this is overriding the assertFn and adding `true` as the last arg? I understand it in the context of this PR, but it would be useful for future readers of the code.